src/main/java/org/testobject/appium/junit/TestObjectAppiumSuite.java

```diff
@@ -211,20 +200,26 @@ private static TestObject getConfig(Class<?> clazz) {
 		return testobject;
 	}
 
-	private Set<String> getRemoteDeviceIds() {
+	private Set<DataCenterSuite> getDataCenterSuites() {
 		AppiumSuiteResource suiteReportResource = new AppiumSuiteResource(client);
-
-		return suiteReportResource.readSuiteDeviceIds(testObjectSuiteId);
+		Set<DataCenterSuite> dataCenterSuites = suiteReportResource.readDataCenterSuites(testObjectSuiteId);
+		for (DataCenterSuite dataCenterSuite : dataCenterSuites) {
+			if (dataCenterSuite.getDeviceDescriptorIds().isEmpty() == false) {
+				return dataCenterSuites;
```

Comment:
Good catch. I agree, this doesn't make sense. I changed the behaviour in https://github.com/testobject/testobject-appium-java-api/pull/18/commits/45836cba1a9589632d45357a1c31f1c80d9f964a